diff --git a/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js b/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js
index ee40370..c649419 100644
--- a/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js
+++ b/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js
@@ -1,6 +1,6 @@
 import _toConsumableArray from "@babel/runtime/helpers/toConsumableArray";
 /* eslint-disable no-underscore-dangle */
-import React from "react";
+import React, { Fragment } from "react";
 import coreUtils from "@opentripplanner/core-utils";
 import { FormattedMessage, useIntl } from "react-intl";
 import styled from "styled-components";
@@ -162,7 +162,7 @@ var FaresV2Table = function FaresV2Table(_ref) {
         style: {
           textAlign: legPrice ? "right" : "center"
         },
-        title: !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 && intl.formatMessage({
+        title: !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 ? intl.formatMessage({
           description: "Text explaining the transfer discount applied to this fare.",
           id: "otpUi.TripDetails.transferDiscountExplanation"
         }, {
@@ -174,8 +174,8 @@ var FaresV2Table = function FaresV2Table(_ref) {
         })
       }, !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 && /*#__PURE__*/React.createElement(TransferIcon, {
         size: 16
-      }), legPrice ? renderFare(legPrice === null || legPrice === void 0 || (_legPrice$currency2 = legPrice.currency) === null || _legPrice$currency2 === void 0 ? void 0 : _legPrice$currency2.code, legPrice === null || legPrice === void 0 ? void 0 : legPrice.amount) : FailDash));
-      var nextRowIndex = currentRowIndex + 1;
+      }) : null, legPrice ? renderFare(legPrice === null || legPrice === void 0 || (_legPrice$currency2 = legPrice.currency) === null || _legPrice$currency2 === void 0 ? void 0 : _legPrice$currency2.code, legPrice === null || legPrice === void 0 ? void 0 : legPrice.amount) : FailDash));
+      const nextRowIndex = currentRowIndex + 1;
       if (!rows[nextRowIndex]) {
         rows[nextRowIndex] = [];
       }
diff --git a/node_modules/@opentripplanner/trip-details/lib/components/fares-v2-table.js b/node_modules/@opentripplanner/trip-details/lib/components/fares-v2-table.js
index c3aa5f1..fd4c5b1 100644
--- a/node_modules/@opentripplanner/trip-details/lib/components/fares-v2-table.js
+++ b/node_modules/@opentripplanner/trip-details/lib/components/fares-v2-table.js
@@ -158,7 +158,7 @@ const FaresV2Table = ({
         style: {
           textAlign: legPrice ? "right" : "center"
         },
-        title: !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 && intl.formatMessage({
+        title: !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 ? intl.formatMessage({
           description: "Text explaining the transfer discount applied to this fare.",
           id: "otpUi.TripDetails.transferDiscountExplanation"
         }, {
@@ -194,9 +194,7 @@ const FaresV2Table = ({
   });
   return /*#__PURE__*/_react.default.createElement(StripedTable, null, /*#__PURE__*/_react.default.createElement(_styled.InvisibleA11yCaption, null, /*#__PURE__*/_react.default.createElement(_reactIntl.FormattedMessage, {
     id: "otpUi.TripDetails.transitFareTable"
-  })), rows.map((r, index) => /*#__PURE__*/_react.default.createElement("tr", {
-    key: index
-  }, r)));
+  })), /*#__PURE__*/_react.default.createElement("tbody", null, rows.map((r, rowIndex) => /*#__PURE__*/_react.default.createElement("tr", {key: "row-".concat(rowIndex)}, r.map((cell, cellIndex) => /*#__PURE__*/_react.default.createElement(_react.default.Fragment, {key: "cell-".concat(rowIndex, "-").concat(cellIndex)}, cell)))));
 };
 var _default = exports.default = FaresV2Table;
 //# sourceMappingURL=fares-v2-table.js.map
\ No newline at end of file
