diff --git a/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js b/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js
index ee40370..4f4e0e0 100644
--- a/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js
+++ b/node_modules/@opentripplanner/trip-details/esm/components/fares-v2-table.js
@@ -162,7 +162,7 @@ var FaresV2Table = function FaresV2Table(_ref) {
         style: {
           textAlign: legPrice ? "right" : "center"
         },
-        title: !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 && intl.formatMessage({
+        title: !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 ? intl.formatMessage({
           description: "Text explaining the transfer discount applied to this fare.",
           id: "otpUi.TripDetails.transferDiscountExplanation"
         }, {
@@ -171,10 +171,10 @@ var FaresV2Table = function FaresV2Table(_ref) {
             currencyDisplay: "narrowSymbol",
             style: "currency"
           })
-        })
-      }, !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 && /*#__PURE__*/React.createElement(TransferIcon, {
+        }) : undefined
+      }, /*#__PURE__*/React.createElement(React.Fragment, null, !Number.isNaN(originalAmount) && originalAmount > 0 && index > 0 && /*#__PURE__*/React.createElement(TransferIcon, {
         size: 16
-      }), legPrice ? renderFare(legPrice === null || legPrice === void 0 || (_legPrice$currency2 = legPrice.currency) === null || _legPrice$currency2 === void 0 ? void 0 : _legPrice$currency2.code, legPrice === null || legPrice === void 0 ? void 0 : legPrice.amount) : FailDash));
+      })), legPrice ? renderFare(legPrice === null || legPrice === void 0 || (_legPrice$currency2 = legPrice.currency) === null || _legPrice$currency2 === void 0 ? void 0 : _legPrice$currency2.code, legPrice === null || legPrice === void 0 ? void 0 : legPrice.amount) : FailDash));
       var nextRowIndex = currentRowIndex + 1;
       if (!rows[nextRowIndex]) {
         rows[nextRowIndex] = [];
