--- a/node_modules/@opentripplanner/building-blocks/src/dropdown/index.tsx
+++ b/node_modules/@opentripplanner/building-blocks/src/dropdown/index.tsx
@@ -124,7 +124,7 @@ const Dropdown = ({
           aria-labelledby={listLabel ? undefined : `${id}-label`}
           id={id}
           onClick={toggleOpen}
           alignLeft={alignMenuLeft}
-          role={isList && "listbox"}
+          role={isList ? "listbox" : undefined}
           tabIndex={-1}
         >
           {children}
